azure/utils.go

```diff
@@ -62,3 +62,9 @@ func toLower(_ context.Context, d *transform.TransformData) (interface{}, error)
 	valStr := types.SafeString(d.Value)
 	return strings.ToLower(valStr), nil
 }
+
+func idToAccountName(ctx context.Context, d *transform.TransformData) (interface{}, error) {
```

Comment:
Are we using this function in any other tables?
If not, move this function to current table.